docs/source/notes/large_scale_deployments.rst

```diff
@@ -27,7 +27,7 @@ across the entire set of machines.
 
 New callbacks for any operator invocation can be added with
 ``torch::autograd::profiler::addGlobalCallback``. Hooks will be called with
-``torch::autograd::profiler::RecordFunction`` struct that describes invocation
+``at::RecordFunction`` struct that describes invocation
```

Comment:
for public C++ API I think we're trying to use `torch::` namespace. So maybe add `using` somewhere (if we haven't already) and refer to it at `torch::RecordFunction` in the docs?